README.md

```diff
@@ -69,7 +71,61 @@ func main() {
 }
 ```
 
-More documentation will be coming as the API stabilises! Things are still in flux.
+### Sending SMS
+
+```golang
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/nexmo-community/nexmo-go"
+)
+
+func main() {
+	auth := nexmo.NewAuthSet()
+	auth.SetAPISecret(API_KEY, API_SECRET)
+
+	client := nexmo.NewClient(http.DefaultClient, auth)
+	var smsReq nexmo.SendSMSRequest
+	smsReq.From = FROM_NUMBER
+	smsReq.To = TO_NUMBER
+	smsReq.Text = "This message comes to you from Nexmo via Golang"
```

Comment:
This can be written as a struct literal:

```golang
smsReq := nexmo.SendSMSRequest {
	From: FROM_NUMBER,
	To: TO_NUMBER,
	Text: "This message comes to you from Nexmo via Golang",
}
```

Reply:
That is a bit nicer, thanks!